server/app/views/applicant/NavigationFragment.html

```diff
@@ -39,7 +39,6 @@
   id="debug-content-modal-button"
   href="#debug-content-modal"
   data-open-modal
-  aria-controls="debug-content-modal"
```

Comment:
this should stay :) The a11y error will go away if you add the modal container back